lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -78,6 +78,16 @@
                 </adf-categories-management>
             </div>
         </ng-container>
+        <ng-container *ngIf="customPanels">
+            <ng-container *ngFor="let customPanel of customPanels">
```

Comment:
Maybe instead of two ng-containers - one with ngIf and one with ngFor, it would be better to initialize customPanels with empty array as default so then ngFor will render nothing and we don't need additional ng-container for ngIf?

Reply:
Good point, reduced to one container